artifact/v3/scripts/executor.hpp

```diff
@@ -71,15 +71,15 @@ class ScriptRunner {
 public:
 	ScriptRunner(
 		events::EventLoop &loop,
-		chrono::seconds state_script_timeout,
+		chrono::milliseconds script_timeout,
```

Comment:
Q: Would it not be better to keep it as is, in seconds?

Reply:
Yes, I would like to. Can you help me [here](https://github.com/mendersoftware/mender/pull/1412#discussion_r1348307973)?

Reply:
Shit. Maybe ur right. This was harder than I thought.

Unsure what to do here. I would kinda like the production code to be in seconds, but then again slow tests.

No hard opinions really. Do what you see fit here :)

Reply:
Being Friday, I will try the template way. I'll put it on a separate commit so that we can decide to drop.

Reply:
I failed.

Reply:
No worries, let's keep templates out of this anywho.

`milliseconds` is fine, let's keep it and move on :)